txscript/script.go

```diff
@@ -335,6 +356,36 @@ func calcHashPrevOuts(tx *wire.MsgTx) chainhash.Hash {
 	return chainhash.DoubleHashH(b.Bytes())
 }
 
+func calcHashUtxos(tx *wire.MsgTx, utxoCache *UtxoCache) chainhash.Hash {
+	var b bytes.Buffer
+
+	for i, _ := range tx.TxIn {
+		utxo, err := utxoCache.GetEntry(i)
+		if err == nil { // TODO have better error handling
```

Comment:
Another TODO, any ideas on how we can handle errors better here, even if it is just debug logging?